hackathon_site/dashboard/frontend/src/components/dashboard/DashCard/DashCard.module.scss

```diff
@@ -0,0 +1,32 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+@import "./../../../assets/abstracts/variables";
+
+.DashCard {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    h2 {
+        margin-bottom: 10px;
+        display: flex;
+        justify-content: left;
```

Comment:
You can delete these two. There was a `text-align: center` in App.scss that was making it like that. It'll be removed in my PR.

Reply:
Done!